appyters/RNA_Level_Functional_Predictions/templates/form.j2

```diff
@@ -0,0 +1,76 @@
+{% extends "biojupies/form.j2" %}
+
+{% block head %}
+{{ super() }}
+{% endblock %}
+
+{% block style %}
+{{ super() }}
+<style>
+
+body {
+  font: 16px Arial;  
+  margin-top: 10%;
+  text-align: center;
+}
+.autocomplete {
+  position: relative;
+  display: inline-block;
+}
+
+input {
+  border: 1px solid transparent;
+  background-color: #f1f1f1;
+  padding: 10px;
+  font-size: 16px;
+}
+
+input[type=text] {
+  background-color: #f1f1f1;
+  width: 100%;
+}
+
+input[type=submit] {
+  background-color: DodgerBlue;
+  color: #fff;
+  cursor: pointer;
+}
+
+.autocomplete-items {
+  position: absolute;
+  border: 1px solid #d4d4d4;
+  border-bottom: none;
+  border-top: none;
+  z-index: 99;
+  /*position the autocomplete items to be the same width as the container:*/
+  top: 100%;
+  left: 0;
+  right: 0;
+}
+
+.autocomplete-items div {
+  padding: 10px;
+  cursor: pointer;
+  background-color: #fff; 
+  border-bottom: 1px solid #d4d4d4; 
+}
+
+.autocomplete-items div:hover {
+  background-color: #e9e9e9; 
+}
+
+.autocomplete-active {
+  background-color: DodgerBlue !important; 
+  color: #ffffff; 
+}
+</style>
+{% endblock %}
+
+{% block body %}
+{{ super() }}
+{% endblock %}
+
+{% block script %}
+{{ super() }}  
+<script src="{{ url_for('static', filename='AutocompleteField.js') }}" name="gene_input" file_path="TCGA.json"> </script>
```

Comment:
Please remove `name` and `file_path`, this information should be extracted from fields on your page.

Consider the following field.j2:

```
<input id={{this.args.name}} class='form-control' type="text" name={{this.args.name}} data-target={{ this.args.file_path }} placeholder="Search Gene Symbol" /> 
```

Your script can then instantiate itself for all fields by looping through all auto-complete inputs:
```
require(['jquery'], function ($) {
  $(document).ready(function() {
    $('.autocomplete').each(function () { // this runs for *all* auto complete fields
      var name = $(this).attr('name')
      var file_path = $(this).attr('data-target')
      // your auto complete script instantiation
    })
  })
})
```

Then you specify `TCGA.json` when you create the field in python:

```
value = {{ AutoCompleteField(name='gene_input', file_path='TCGA.json') }}
```

Note that with these changes, you can then add another auto complete field with a different set of elements defined by a different json.